.precious.toml

```
@@ -0,0 +1,53 @@
+[commands.prettier-scripts]
+type = "both"
+cmd = [
+  "npx",
+  "prettier",
+  "--parser",
+  "typescript"
+]
```

I wonder if it makes more sense to use `npm run <command>` here and below. That way we don't need to update both this file and the `package.json` if we ever wanted to make changes to either command.

```suggestion
cmd = [
  "npm",
  "run",
  "prettier:ts"
]
```

---

I see your point about things potentially going out of sync.   Although `npx` is useful in the (rare) case that a dev decides not to run `npm install`.  I suppose both cases - things going out of sync and a dev not running `npm install` are both quite rare, so to me it's a coin flip